src/lang/id_ID.rc

```diff
@@ -0,0 +1,1583 @@
+
+IDD_EGA DIALOG 0, 0, 400, 225
+CAPTION "Bahasa Pemrograman EGA"
+STYLE WS_POPUPWINDOW | WS_CAPTION | WS_THICKFRAME
+FONT 9, "MS Shell Dlg"
+{
+    GROUPBOX "Konsol EGA", grp1, 5, 5, 390, 195
+    EDITTEXT edt1, 10, 20, 380, 170, ES_READONLY | ES_AUTOVSCROLL | ES_MULTILINE | WS_VSCROLL | NOT WS_TABSTOP
+    RTEXT "EGA>", stc1, 10, 208, 20, 10
+    COMBOBOX cmb1, 35, 205, 300, 300, CBS_NOINTEGRALHEIGHT | CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    DEFPUSHBUTTON "&Masukkan", IDOK, 345, 205, 50, 15
+}
+
+IDD_REPLACEFONTS DIALOG 0, 0, 185, 180
+CAPTION "Mengganti Fon Dialog"
+STYLE DS_CENTER | DS_MODALFRAME | WS_POPUPWINDOW | WS_CAPTION
+FONT 9, "MS Shell Dlg"
+{
+    LTEXT "Font #&1:", -1, 5, 7, 44, 12
+    COMBOBOX cmb1, 55, 5, 125, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    ICON IDI_ARROW, -1, 30, 25, 0, 0
+    COMBOBOX cmb2, 55, 30, 125, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    LTEXT "Font #&2:", -1, 5, 57, 44, 12
+    COMBOBOX cmb3, 55, 55, 125, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    ICON IDI_ARROW, -1, 30, 75, 0, 0
+    COMBOBOX cmb4, 55, 80, 125, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    LTEXT "Font #&3:", -1, 5, 105, 44, 12
+    COMBOBOX cmb5, 55, 105, 125, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    ICON IDI_ARROW, -1, 30, 125, 0, 0
+    COMBOBOX cmb6, 55, 130, 125, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    DEFPUSHBUTTON "OK", IDOK, 75, 160, 50, 14
+    PUSHBUTTON "Batal", IDCANCEL, 130, 160, 50, 14
+    PUSHBUTTON "&Set Ulang", psh1, 5, 160, 55, 14
+}
+
+IDD_DROPDOWNPOPUP DIALOG 0, 0, 120, 150
+CAPTION " "
+STYLE WS_POPUP | WS_DLGFRAME | NOT WS_BORDER
+FONT 9, "MS Shell Dlg"
+{
+    LISTBOX lst1, 0, 0, 120, 150, LBS_WANTKEYBOARDINPUT | LBS_NOINTEGRALHEIGHT | LBS_HASSTRINGS | WS_VSCROLL | WS_TABSTOP, WS_EX_CLIENTEDGE
+}
+
+IDD_DFMSETTINGS DIALOG 0, 0, 210, 135
+CAPTION "Pengaturan Delphi DFM"
+STYLE DS_CENTER | DS_MODALFRAME | WS_POPUPWINDOW | WS_CAPTION
+FONT 9, "MS Shell Dlg"
+{
+    LTEXT "Pengaturan berikut relevan ke DFM:", -1, 35, 5, 171, 20
+    ICON IDI_DFMICON, -1, 5, 5, 0, 0
+    RTEXT "&Codepage:", -1, 6, 32, 70, 12
+    COMBOBOX cmb1, 85, 30, 120, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    AUTOCHECKBOX "&Gunakan komentar raw string", chx1, 5, 50, 200, 14
+    AUTOCHECKBOX "Jangan gunakan &Unicode dan UTF-8", chx2, 5, 70, 200, 14
+    DEFPUSHBUTTON "OK", IDOK, 80, 115, 60, 14
+    PUSHBUTTON "Batal", IDCANCEL, 145, 115, 60, 14
+}
+
+//////////////////////////////////////////////////////////////////////////////
+// RT_VERSION
+
+1 VERSIONINFO
+FILEVERSION     5, 4, 4, 0
+PRODUCTVERSION  5, 4, 4, 0
+FILEOS          0x40004
+FILETYPE        0x1
+{
+    BLOCK "StringFileInfo"
+    {
+        BLOCK "040904E4"
```

Comment:
Also these values.

Reply:
i found the charset. it's 4b0 in a web reference